src/screens/about/index.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+class About extends React.Component {
+  render() {
+    return (
+      <div>
+        {/* the issues and source links will live in the header*/}
+        <a href="https://github.com/FormidableLabs/component-playground">Source Code on GitHub</a>
+        <br/>
+        <a href="https://github.com/FormidableLabs/component-playground/issues">Report an Issue</a>
```

Comment:
They will, and they'll have the header, I just wanted to add them to the raw html in case they were helpful. But I'll take them out and then merge!